lib/teslamate/mqtt/mqtt.ex

```diff
@@ -27,13 +27,25 @@ defmodule TeslaMate.Mqtt do
   defp config do
     auth = Application.get_env(:teslamate, :mqtt)
 
-    [
-      user_name: Keyword.get(auth, :username),
-      password: Keyword.get(auth, :password),
-      server: {Tortoise.Transport.Tcp, host: Keyword.get(auth, :host), port: 1883},
-      handler: {Tortoise.Handler.Logger, []},
-      subscriptions: []
-    ]
+    if(Keyword.get(auth, :ssl) == "false") do
+      [
+        user_name: Keyword.get(auth, :username),
+        password: Keyword.get(auth, :password),
+        server: {Tortoise.Transport.Tcp, host: Keyword.get(auth, :host), port: 1883},
+        handler: {Tortoise.Handler.Logger, []},
+        subscriptions: []
+      ]
+    else
+      [
+        user_name: Keyword.get(auth, :username),
+        password: Keyword.get(auth, :password),
+        server:
+          {Tortoise.Transport.SSL,
+           host: Keyword.get(auth, :host), port: 8883, verify: :verify_none},
```

Comment:
I think one would expect security be default i.e. certificates should be verified when `MQTT_SSL` is enabled.

Do you use a self-signed certificate? Or could we add a CA store with the `:cacertfile` option?

Reply:
Yes, I use a self-signed certificate, hence the verify_none option. But a CA store option would work as well.

Reply:
Gotcha! Then let's add a CA store and make`:verify_none` optional e.g. with another env var? I'll try to integrate  https://github.com/ericmj/castore  later.

Reply:
Perfect! Looking forward to test.

Thanks!